src/components/submission/decrypt.vue

```diff
@@ -34,9 +34,11 @@ except according to the terms contained in the LICENSE file.
               {{ $t('field.removeGroupNames') }}
             </label>
           </div>
-          <div class="checkbox">
-            <label>
-              <input v-model="deletedFields" type="checkbox">
+          <div class="checkbox"
+            :class="{ disabled: deletedFieldsTitle != null }">
+            <label :title="deletedFieldsTitle">
+              <input v-model="deletedFields" type="checkbox"
+                :disabled="deletedFieldsTitle != null">
```

Comment:
The way this is implemented is very similar to the `splitSelectMultiples` checkbox above.